app/pdfs/master_skill.rb

```diff
@@ -0,0 +1,44 @@
+class MasterSkill < Prawn::Document
```

Comment:
I should really update all of the issues to drop this language of 'master skills' because I think it's causing confusion. "Master skills" referred to essentially what is a `FrameworkStandard` in this codebase.

I think this object should just be an 'OccupationStandardPdf`.

Reply:
Okay great, I was feeling like it was going to cause confusion down the road since there are also plain old skills. I will make that change.

Reply:
@pcushing on the note that Master Skill == FrameworkStandard --  should only FrameworkStandard types be able to have a pdf generated?  Or all should be allowed.

Reply:
@littleforest All should be allowed.